### PR TITLE
docs: express middleware

### PR DESCRIPTION
**Pull request by madina0801**
Issue #30 
Please, let me know if you'd like to have a more clear documentation and where it should be located :)

### CODE REVIEW

File: documentation/content/sdk/node/get-express-middleware.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# `getExpressMiddleware` Function Documentation
+
+The `getExpressMiddleware` function is a utility function that returns an Express middleware with customizable options. This middleware is particularly useful for integrating Prici SDK functionality into Express applications.
+
+## Usage
+
+```javascript
+const express = require('express');
+const { getExpressMiddleware } = require('your-module');
```
Comment (davidmeirlevy) [inline documentation/content/sdk/node/get-express-middleware.md:9]: It's coming from "@prici/sdk/express"

File: documentation/content/sdk/node/get-express-middleware.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+const app = express();
+
+const middlewareOptions = {
+  sdk: // Your Prici SDK instance,
+  fieldId: // Optional: Specify the fieldId,
+  errorMessage: // Optional: Custom error message, defaults to 'reached limit',
+  incrementAmount: // Optional: Specify the increment amount,
```
Comment (davidmeirlevy) [inline documentation/content/sdk/node/get-express-middleware.md:17]: There are also "getX" for every property, that allows a sun functions for each property.

**Review by davidmeirlevy — CHANGES_REQUESTED** [review]

**Comment by davidmeirlevy** [comment]
Thanks for the docs.

Please fix the two comments, and also add a link to this page on .vitepress/config.js

**Comment by madina0801** [comment]
Written more detailed docs on middleware options, example usage and updated config :)

**Comment by davidmeirlevy** [comment]
Thanks!

Looks good.

**Pull request merged**